generator/internal/format_method_comments.cc

```diff
@@ -123,16 +123,16 @@ std::string FormatMethodComments(
     GCP_LOG(FATAL) << __FILE__ << ":" << __LINE__ << ": " << method.full_name()
                    << " no leading_comments to format";
   }
-  std::string doxygen_formatted_function_comments = absl::StrReplaceAll(
-      method_source_location.leading_comments,
-      {
-          {"\n", "\n  ///"},
-          {"$", "$$"},
-          // TODO(#12190) - track whether this substitution is used.           \
```

Comment:
Is there a problem, officer?

Reply:
https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html

```
-Wcomment
-Wcomments
Warn whenever a comment-start sequence ‘/*’ appears in a ‘/*’ comment, or whenever a backslash-newline appears in a ‘//’ comment. This warning is enabled by -Wall.
```

That means `-Wall -Werror` would not go well.

Reply:
It was easy to miss the trailing `\` here.

Reply:
And easy to remove the comment on the next line and get some live code commented-out.

Reply:
FWIW, I did not want the "TODO" on the same line as "From logging/v2", because they are separate thoughts. And the `\` was my way around it.

Reply:
Ah, sorry.  You may need to insert an extra `//` line.